course_content/notebooks/matplotlib_intro.ipynb

```diff
@@ -81,25 +68,15 @@
-       "text": [
-        "<matplotlib.figure.Figure at 0x43c21d0>"
-       ]
-      }
-     ],
-     "prompt_number": 3
+     "outputs": []
```

Comment:
@djkirkham @pp-mo @dkillick I think we need to agree on a strategy here before I start chopping cell outputs in and out of the commits.